packages/temen/src/conformsTo/index.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * 'object'가 인라인 함수 'source'와 일치하는지 확인
+ * properties of `source` with the corresponding property values of `object`.
+ *
+ *
+ * @param {Object} object 검사할 오브젝트
+ * @param {Object} source 오브젝트의 프로퍼티를 검증할 오브젝트
+ * @returns {boolean} 'object'가 일치하면 'true'를 반환하고, 그렇지 않으면 'false'를 반환
+ * @example
+ *
+ * const object = { 'a': 1, 'b': 2 }
+ *
+ * conformsTo(object, { 'b': function(n) { return n > 1 } })
+ * // => true
+ *
+ * conformsTo(object, { 'b': function(n) { return n > 2 } })
+ * // => false
```

Comment:
선생님 마크다운 추가해주셔야합니다 :whip: :whip:

```suggestion
 * ```ts
 * const object = { 'a': 1, 'b': 2 }
 *
 * conformsTo(object, { 'b': function(n) { return n > 1 } })
 * // => true
 *
 * conformsTo(object, { 'b': function(n) { return n > 2 } })
 * // => false
 * ```
```

Reply:
기억하자 기억하자 기억하자..

Reply:
써제션감사합니당